src/main/java/mate/academy/spring/model/dto/response/OrderResponseDto.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.spring.model.dto.response;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class OrderResponseDto {
+    private Long id;
+    private List<Long> ticketsIds;
+    private LocalDateTime orderDate;
```

Comment:
let's return in response some simple data types, like primitives, wrappers and String. smth like LocalDateTime will be hard to parse